docs/design/trace_points/runtime_trace_points.md

```diff
@@ -77,7 +90,9 @@ erDiagram
     rmw_take ||--|| callback_start: tid
     dispatch_intra_process_subscription_callback ||--|| callback_start: callback
     dispatch_subscription_callback ||--|| callback_start: callback
-
+    rclcpp_intra_publish ||--|| rclcpp_ring_buffer_enqueue: tid
+    rclcpp_ring_buffer_enqueue ||--|| rclcpp_ring_buffer_dequeue: buffer, index
+    rclcpp_ring_buffer_dequeue ||--|| callback_start: tid
```

Comment:
I think this change is correct, but it seems [the result](https://tier4.github.io/caret_doc/pr-207/design/trace_points/runtime_trace_points/) is broken. Can you check it?

Reply:
Thanks for reviewing.
fixed in [830f635](https://github.com/tier4/caret_doc/pull/207/commits/830f635c9315f220101c030e70b771f7ab255227).